src/components/modal/CdrModal.vue

```diff
@@ -14,14 +23,16 @@ import mapClasses from '../../utils/mapClasses';
 
 /** Disruptive, action-blocking overlays used to display important information */
 defineOptions({
-  name: 'CdrModal'
+  name: 'CdrModal',
+  inheritAttrs: false,
```

Comment:
Only actual change, the rest is formatting, just making more friendly for split-screen viewing.

Reply:
Yep, good catch. This is a bug introduced when we migrated to Vue 3.